static/src/stylesheets/module/_media.scss

```diff
@@ -36,3 +36,9 @@ Nicole Sullivan's OOCSS CSS markup extraction pattern.
 .media__img img {
     display: block;
 }
+
+.media__container--hidden { display: none; }
+.media__placeholder--active { display: block; }
+
+.media__container--active { display: block; }
+.media__placeholder--hidden { display: none; }
```

Comment:
Thanks good point @sammorrisdesign and I did consider that but it seems to be designed to hide it visibly but keep it on the page (e.g. for screen readers)  I'm not sure if that's what I need as I am substituting two equivalents completely.  What do you think?  When would we use that over display: none in general?